svcacct doc updates from RELEASE.2023-05-18T16-59-00Z (#868)
minio/docs: source/reference/minio-mc-admin/mc-admin-user-svcacct-add.rst

.. mc-cmd:: --access-key
   :optional:

   A string to use as the access key for this account.
   Omit to let MinIO autogenerate a random 20 character value.

   Access Key names *must* be unique across all users.

.. mc-cmd:: --comment
   :optional:

>    .. versionchanged:: RELEASE.2023-05-18T16-59-00Z

Author: Should there be both `versionadded` and `versionchanged` for this?

Reviewer: I'd keep both.
Versionadded means it won't be present at all in releases older than that release.
Versionchanged means something about the functionality is different from that release on than what users might be accustomed to in an older version.

Reviewer: I think I'd just keep the changed directive, but then say something like
```suggestion
   .. versionchanged:: RELEASE.2023-05-18T16-59-00Z
      Replaced by :mc-cmd:`~mc admin user svcacct add --description` and :mc-cmd:`~mc admin user svcacct add --name`.
      
      Originally added in version RELEASE.2023-01-28T20-29-38Z.

```

Reviewer: Agree on dropping the added directive, to be clear.

   This option has been removed.
   Use ``--description`` or ``--name`` instead.

.. mc-cmd:: --description
   :optional:

   .. versionadded:: RELEASE.2023-05-18T16-59-00Z

   Add a description for the service account.
   For example, you might specify the reason the service account exists.

.. mc-cmd:: --name
   :optional:

   .. versionadded:: RELEASE.2023-05-18T16-59-00Z

>    Add a friendly name for the service account.

Author: This text matches what is in `--help`. 

Is it worth adding an example ("Test account for xyz team" ?) or is `friendly` sufficiently understood without additional context?

Reviewer: Maybe
```suggestion
   Add a human-readable name for the service account.
```

.. mc-cmd:: --secret-key
   :optional:

   The secret key to associate with the new account.

>    Omit to let MinIO autogenerate a random 40 character value.

Reviewer: ```suggestion
   Omit to let MinIO autogenerate a random 40-character value.
```
